go.mod

```diff
@@ -5,7 +5,10 @@ go 1.22
 toolchain go1.22.0
 
 require (
-	github.com/Layr-Labs/eigenda v0.8.4
+	// this is the latest commit of https://github.com/Layr-Labs/eigenda/pull/828
+	// TODO: merge upstream first and the use the latest commit of the main branch
+	github.com/Layr-Labs/eigenda v0.8.5-0.20241025144504-69ff616c1e5b
```

Comment:
TODO: need to merge eigenda PR first. https://github.com/Layr-Labs/eigenda/pull/828